content/2022-05-19-leave-the-yaml-at-home.md

```diff
@@ -0,0 +1,185 @@
+---
+title: "Leave the YAML at Home!"
+slug: "2022-05-19-leave-the-yaml-at-home"
+description: "Did you know Brigade involves a minimal amount of YAML?"
+date: "2022-05-19"
+author: "Kent Rancourt"
+summary: "We hear people aren't always crazy about YAML. Have you met Brigade?"
+---
+
+Hello again, Brigadiers!
+
+We've heard from time to time that people aren't always so crazy about YAML --
+and that's putting it nicely. I don't personally harbor such strong anti-YAML
+sentiment, but the critics are out there and as critics are wont to be, they're
+quite vocal. If you dare to search for "yaml sucks" in your favorite search
```

Comment:
```suggestion
sentiment, but critics are out there, and they're quite vocal. If you dare to search for "yaml sucks" in your favorite search
```

Reply:
Yup.... "wont" is too archaic. Changing.